From 9c1d2ac267bc37484b6f3489a4cd4f8642f9f533 Mon Sep 17 00:00:00 2001
From: mrzhuzhe <466648706@qq.com>
Date: Wed, 15 Jul 2026 14:22:33 +0800
Subject: [PATCH 2/2] add-support-hdl

---
 scripts/adi_env.tcl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/adi_env.tcl b/scripts/adi_env.tcl
index 53eae54d1..fb5b6eb9a 100644
--- a/scripts/adi_env.tcl
+++ b/scripts/adi_env.tcl
@@ -15,7 +15,7 @@ if [info exists ::env(ADI_GHDL_DIR)] {
 }
 
 # Define the supported tool version
-set required_vivado_version "2023.2"
+set required_vivado_version "2025.2"
 if {[info exists ::env(REQUIRED_VIVADO_VERSION)]} {
   set required_vivado_version $::env(REQUIRED_VIVADO_VERSION)
 } elseif {[info exists REQUIRED_VIVADO_VERSION]} {
-- 
2.54.0

